pymc4/distributions/continuous.py

```diff
@@ -337,26 +314,22 @@ class ChiSquared(PositiveContinuousDistribution):
 
     Parameters
     ----------
-    nu : int
-        Degrees of freedom (nu > 0).
+    df : int
+        Degrees of freedom (df > 0).
 
     Developer Notes
     ----------------
-    Parameter mappings to TensorFlow Probability are as follows:
-
-    - nu: df
-
     The ChiSquared distribution name is copied over from PyMC3 for continuity. We map it to the
     Chi2 distribution in TensorFlow Probability.
```

Comment:
According to this, we're calling it `ChiSquared` instead of `Chi2` to maintain continuity. Since nothing else is maintaining continuity, I suggest we also call this `Chi2`. Thoughts?

Reply:
I've gone ahead and done this in https://github.com/pymc-devs/pymc4/pull/155/commits/520459ff6cfde96bd8a0ab0f263749c6144adef9

I can revert that if there are objections 😄

Reply:
Agreed.